### ISSUE TITLE
Need a way to get album details using albumid (token)

### ISSUE THREAD

**Issue by arsaboo** [dmdhrumilmistry/MusicAPy#5]
So, I am trying to manually create an `identifier` using the `albumid`, but it is not working. See the code below:
```python
link = "https://www.jiosaavn.com/album/tu-jhoothi-main-makkaar/Zk6,htMIngg_"`
identifier = jiosaavn.create_identifier(link, 'album')
print(identifier)
print(jiosaavn.get_album_details(identifier))
```
This produces the following output:
```
{'type': 'link', 'value': 'Zk6,htMIngg_'}
{'title': 'Tu Jhoothi Main Makkaar', 'name': 'Tu Jhoothi Main Makkaar', 'year': '2023', 'release_date': '2023-03-16', 'primary_artists': 'Pritam', 'primary_artists_id': '456323', 'albumid': '43600311', 'perma_url': 'https://www.jiosaavn.com/album/tu-jhoothi-main-makkaar/Zk6,htMIngg_', 'image': 'http://c.saavncdn.com/903/Tu-Jhoothi-Main-Makkaar-Hindi-2023-20230316165419-150x150.jpg', 'songs': [{'id': 'ESqTThi-', 'type': '', 'song': 'Tere Pyaar Mein', 'album': 'Tu Jhoothi Main Makkaar', 'year': '2023', 'music': 'Pritam', 'music_id': '456323', 'primary_artists': 'Pritam, Arijit Singh, Nikhita Gandhi, Amitabh Bhattacharya', 'primary_artists_id': '456323, 459320, 745460, 458681', 'featured_artists': '', 'featured_artists_id': '', 'singers': 'Pritam, Arijit Singh, Nikhita Gandhi, Amitabh Bhattacharya', 'starring': '', 'image': 'http://c.saavncdn.com/903/Tu-Jhoothi-Main-Makkaar-Hindi-2023-20230316165419-150x150.jpg', 'label': '', 'albumid': '43600311', 'language': 'hindi', 'origin': 'album', 'play_count': '5563883', 'is_ringtone_available': False, 'copyright_text': '℗ 2023 Super Cassettes Industries Private Limited', '320kbps': 'true', 'is_dolby_content': False, 'explicit_content': 0, 'has_lyrics': 'false', 'lyrics_snippet': '', 'encrypted_media_url': 'ID2ieOjCrwfgWvL5sXl4B1ImC5QfbsDygeJVTrKUhFJx5+h7xMaEgJS+oHt6HNmppklXa/mBOGyRmSy64Jrnzxw7tS9a8Gtq', 'encrypted_media_path': 'NMKyboFo/Fi0VFqJf5Q3Rw==', 'media_preview_url': 'https://preview.saavncdn.com/903/643f0dffc7bfbec39b93cf3c35c1f90d_96_p.mp4', 'perma_url': 'https://www.jiosaavn.com/song/tere-pyaar-mein/NTsaZSBYXh4', 'album_url': 'https://www.jiosaavn.com/album/tu-jhoothi-main-makkaar/Zk6,htMIngg_', 'duration': '266', 'rights': {'code': 1, 'reason': 'Pro Only', 'cacheable': True, 'delete_cached_object': False}, 'webp': True, 'disabled': 'true', 'disabled_text': 'Pro Only', 'starred': 'false', 'artistMap': {'Pritam': '456323', 'Arijit Singh': '459320', 'Nikhita Gandhi': '745460', 'Amitabh Bhattacharya': '458681'}, 'release_date': '2023-03-16', 'vcode': '010910091863721', 'vlink': 'https://jiotunepreview.jio.com/content/Converted/010910091819687.mp3', 'triller_available': False, 'label_url': '/label/-albums/6DLuXO3VoTo_'}, {'id': 'pvUfIxm9', 'type': '', 'song': 'Pyaar Hota Kayi Baar Hai', 'album': 'Tu Jhoothi Main Makkaar', 'year': '2023', 'music': 'Pritam', 'music_id': '456323', 'primary_artists': 'Pritam, Arijit Singh, Amitabh Bhattacharya, Charan', 'primary_artists_id': '456323, 459320, 458681, 460577', 'featured_artists': '', 'featured_artists_id': '', 'singers': 'Pritam, Arijit Singh, Amitabh Bhattacharya, Charan', 'starring': '', 'image': 'http://c.saavncdn.com/903/Tu-Jhoothi-Main-Makkaar-Hindi-2023-20230316165419-150x150.jpg', 'label': '', 'albumid': '43600311', 'language': 'hindi', 'origin': 'album', 'play_count': '3614661', 'is_ringtone_available': False, 'copyright_text': '℗ 2023 Super Cassettes Industries Private Limited', '320kbps': 'true', 'is_dolby_content': False, 'explicit_content': 0, 'has_lyrics': 'false', 'lyrics_snippet': '', 'encrypted_media_url': 'ID2ieOjCrwfgWvL5sXl4B1ImC5QfbsDym0wRjMYIVANgZUqsnGnspcEu0oXEEpxussGqIXK/CIrmkYZzSNMmxhw7tS9a8Gtq', 'encrypted_media_path': 'NMKyboFo/Fi0VFqJf5Q3Rw==', 'media_preview_url': 'https://preview.saavncdn.com/903/be34fee24a5177160ed52356406f7a67_96_p.mp4', 'perma_url': 'https://www.jiosaavn.com/song/pyaar-hota-kayi-baar-hai/AB4,Vz1IWgo', 'album_url': 'https://www.jiosaavn.com/album/tu-jhoothi-main-makkaar/Zk6,htMIngg_', 'duration': '216', 'rights': {'code': 1, 'reason': 'Pro Only', 'cacheable': True, 'delete_cached_object': False}, 'webp': True, 'disabled': 'true', 'disabled_text': 'Pro Only', 'starred': 'false', 'artistMap': {'Pritam': '456323', 'Arijit Singh': '459320', 'Amitabh Bhattacharya': '458681', 'Charan': '460577'}, 'release_date': '2023-03-16', 'vcode': '010910091870266', 'vlink': 'https://jiotunepreview.jio.com/content/Converted/010910091826883.mp3', 'triller_available': False, 'label_url': '/label/-albums/6DLuXO3VoTo_'}, {'id': 'sV2Cuese', 'type': '', 'song': 'O Bedardeya', 'album': 'Tu Jhoothi Main Makkaar', 'year': '2023', 'music': 'Pritam', 'music_id': '456323', 'primary_artists': 'Pritam, Arijit Singh, Amitabh Bhattacharya', 'primary_artists_id': '456323, 459320, 458681', 'featured_artists': '', 'featured_artists_id': '', 'singers': 'Pritam, Arijit Singh, Amitabh Bhattacharya', 'starring': '', 'image': 'http://c.saavncdn.com/903/Tu-Jhoothi-Main-Makkaar-Hindi-2023-20230316165419-150x150.jpg', 'label': '', 'albumid': '43600311', 'language': 'hindi', 'origin': 'album', 'play_count': '1760720', 'is_ringtone_available': False, 'copyright_text': '℗ 2023 Super Cassettes Industries Private Limited', '320kbps': 'true', 'is_dolby_content': False, 'explicit_content': 0, 'has_lyrics': 'false', 'lyrics_snippet': '', 'encrypted_media_url': 'ID2ieOjCrwfgWvL5sXl4B1ImC5QfbsDySdsy0kDT51DAJWpiDAOz969fnTwcRliLscYcRYLzozM1Yuc5xyHDZBw7tS9a8Gtq', 'encrypted_media_path': 'NMKyboFo/Fi0VFqJf5Q3Rw==', 'media_preview_url': 'https://preview.saavncdn.com/903/a036e9cb0914abaf15b3c95855f9c5c4_96_p.mp4', 'perma_url': 'https://www.jiosaavn.com/song/o-bedardeya/Az5ZcgFVRFY', 'album_url': 'https://www.jiosaavn.com/album/tu-jhoothi-main-makkaar/Zk6,htMIngg_', 'duration': '313', 'rights': {'code': 1, 'reason': 'Pro Only', 'cacheable': True, 'delete_cached_object': False}, 'webp': True, 'disabled': 'true', 'disabled_text': 'Pro Only', 'starred': 'false', 'artistMap': {'Pritam': '456323', 'Arijit Singh': '459320', 'Amitabh Bhattacharya': '458681'}, 'release_date': '2023-03-16', 'vcode': '010910091901333', 'vlink': 'https://jiotunepreview.jio.com/content/Converted/010910091858172.mp3', 'triller_available': False, 'label_url': '/label/-albums/6DLuXO3VoTo_'}, {'id': 'CAC123DR', 'type': '', 'song': 'Show Me The Thumka', 'album': 'Tu Jhoothi Main Makkaar', 'year': '2023', 'music': 'Pritam', 'music_id': '456323', 'primary_artists': 'Pritam, Amitabh Bhattacharya, Sunidhi Chauhan, Shashwat Singh', 'primary_artists_id': '456323, 458681, 455129, 909299', 'featured_artists': '', 'featured_artists_id': '', 'singers': 'Pritam, Amitabh Bhattacharya, Sunidhi Chauhan, Shashwat Singh', 'starring': '', 'image': 'http://c.saavncdn.com/903/Tu-Jhoothi-Main-Makkaar-Hindi-2023-20230316165419-150x150.jpg', 'label': '', 'albumid': '43600311', 'language': 'hindi', 'origin': 'album', 'play_count': '2612918', 'is_ringtone_available': False, 'copyright_text': '℗ 2023 Super Cassettes Industries Private Limited', '320kbps': 'true', 'is_dolby_content': False, 'explicit_content': 0, 'has_lyrics': 'false', 'lyrics_snippet': '', 'encrypted_media_url': 'ID2ieOjCrwfgWvL5sXl4B1ImC5QfbsDydCEutku7IIqk6jYUG9VFY3n/524YGuwGzDuOuoFmDseYYBD8ZKYBcxw7tS9a8Gtq', 'encrypted_media_path': 'NMKyboFo/Fi0VFqJf5Q3Rw==', 'media_preview_url': 'https://preview.saavncdn.com/903/b57a8d48e48051d1a281b2778ba3c9c7_96_p.mp4', 'perma_url': 'https://www.jiosaavn.com/song/show-me-the-thumka/MykoAEYDc2E', 'album_url': 'https://www.jiosaavn.com/album/tu-jhoothi-main-makkaar/Zk6,htMIngg_', 'duration': '236', 'rights': {'code': 1, 'reason': 'Pro Only', 'cacheable': True, 'delete_cached_object': False}, 'webp': True, 'disabled': 'true', 'disabled_text': 'Pro Only', 'starred': 'false', 'artistMap': {'Pritam': '456323', 'Amitabh Bhattacharya': '458681', 'Sunidhi Chauhan': '455129', 'Shashwat Singh': '909299'}, 'release_date': '2023-03-16', 'vcode': '010910091879246', 'vlink': 'https://jiotunepreview.jio.com/content/Converted/010910091835772.mp3', 'triller_available': False, 'label_url': '/label/-albums/6DLuXO3VoTo_'}, {'id': 'U_-LFOuj', 'type': '', 'song': 'Jaadui', 'album': 'Tu Jhoothi Main Makkaar', 'year': '2023', 'music': 'Pritam', 'music_id': '456323', 'primary_artists': 'Pritam, Jubin Nautiyal, Amitabh Bhattacharya', 'primary_artists_id': '456323, 881158, 458681', 'featured_artists': '', 'featured_artists_id': '', 'singers': 'Pritam, Jubin Nautiyal, Amitabh Bhattacharya', 'starring': '', 'image': 'http://c.saavncdn.com/903/Tu-Jhoothi-Main-Makkaar-Hindi-2023-20230316165419-150x150.jpg', 'label': '', 'albumid': '43600311', 'language': 'hindi', 'origin': 'album', 'play_count': '501141', 'is_ringtone_available': False, 'copyright_text': '℗ 2023 Super Cassettes Industries Private Limited', '320kbps': 'true', 'is_dolby_content': False, 'explicit_content': 0, 'has_lyrics': 'false', 'lyrics_snippet': '', 'encrypted_media_url': 'ID2ieOjCrwfgWvL5sXl4B1ImC5QfbsDypB3w8k1mfhX8yAi/1noofnuBWraN2m1qg1FcllwEllL5TTRAa1Ikxhw7tS9a8Gtq', 'encrypted_media_path': 'NMKyboFo/Fi0VFqJf5Q3Rw==', 'media_preview_url': 'https://preview.saavncdn.com/903/1fcf095000a9f3c6abe54ef2cf93a739_96_p.mp4', 'perma_url': 'https://www.jiosaavn.com/song/jaadui/JTdGfTJ-Qlk', 'album_url': 'https://www.jiosaavn.com/album/tu-jhoothi-main-makkaar/Zk6,htMIngg_', 'duration': '222', 'rights': {'code': 1, 'reason': 'Pro Only', 'cacheable': True, 'delete_cached_object': False}, 'webp': True, 'disabled': 'true', 'disabled_text': 'Pro Only', 'starred': 'false', 'artistMap': {'Pritam': '456323', 'Jubin Nautiyal': '881158', 'Amitabh Bhattacharya': '458681'}, 'release_date': '2023-03-16', 'vcode': '010910091901323', 'vlink': 'https://jiotunepreview.jio.com/content/Converted/010910091857824.mp3', 'triller_available': False, 'label_url': '/label/-albums/6DLuXO3VoTo_'}, {'id': 'PI_vT8xk', 'type': '', 'song': 'Maine Pi Rakhi Hai', 'album': 'Tu Jhoothi Main Makkaar', 'year': '2023', 'music': 'Pritam', 'music_id': '456323', 'primary_artists': 'Pritam, Shreya Ghoshal, Amitabh Bhattacharya, Divya Kumar', 'primary_artists_id': '456323, 455130, 458681, 473376', 'featured_artists': '', 'featured_artists_id': '', 'singers': 'Pritam, Shreya Ghoshal, Amitabh Bhattacharya, Divya Kumar', 'starring': '', 'image': 'http://c.saavncdn.com/903/Tu-Jhoothi-Main-Makkaar-Hindi-2023-20230316165419-150x150.jpg', 'label': '', 'albumid': '43600311', 'language': 'hindi', 'origin': 'album', 'play_count': '258514', 'is_ringtone_available': False, 'copyright_text': '℗ 2023 Super Cassettes Industries Private Limited', '320kbps': 'true', 'is_dolby_content': False, 'explicit_content': 0, 'has_lyrics': 'false', 'lyrics_snippet': '', 'encrypted_media_url': 'ID2ieOjCrwfgWvL5sXl4B1ImC5QfbsDywijcuhS9TokUZ88gexojE144Nn/R73NpjUDa0Ny+BNHTfWULjAZBGhw7tS9a8Gtq', 'encrypted_media_path': 'NMKyboFo/Fi0VFqJf5Q3Rw==', 'media_preview_url': 'https://preview.saavncdn.com/903/bd2aad4d1c5a9195213fe5caed891dd1_96_p.mp4', 'perma_url': 'https://www.jiosaavn.com/song/maine-pi-rakhi-hai/ICE0RyAIT1g', 'album_url': 'https://www.jiosaavn.com/album/tu-jhoothi-main-makkaar/Zk6,htMIngg_', 'duration': '247', 'rights': {'code': 1, 'reason': 'Pro Only', 'cacheable': True, 'delete_cached_object': False}, 'webp': True, 'disabled': 'true', 'disabled_text': 'Pro Only', 'starred': 'false', 'artistMap': {'Pritam': '456323', 'Shreya Ghoshal': '455130', 'Amitabh Bhattacharya': '458681', 'Divya Kumar': '473376'}, 'release_date': '2023-03-16', 'vcode': '010910091901338', 'vlink': 'https://jiotunepreview.jio.com/content/Converted/010910091858209.mp3', 'triller_available': False, 'label_url': '/label/-albums/6DLuXO3VoTo_'}], 'modules': {'reco': {'source': 'reco.getAlbumReco', 'position': 1, 'score': '', 'bucket': '', 'scroll_type': 'SS_Basic', 'title': 'You Might Like', 'subtitle': '', 'highlight': '', 'simpleHeader': False, 'noHeader': False, 'source_api': True, 'source_params': {'albumid': '43600311'}, 'view_more': [], 'is_JT_module': False}, 'currentlyTrending': {'source': 'content.getTrending', 'position': 2, 'score': '', 'bucket': '', 'scroll_type': 'SS_Basic', 'title': 'Currently Trending Albums', 'subtitle': '', 'highlight': '', 'simpleHeader': False, 'noHeader': False, 'source_api': True, 'source_params': {'entity_type': 'album', 'entity_language': None}, 'view_more': [], 'is_JT_module': False}, 'topAlbumsFromSameYear': {'source': 'search.topAlbumsoftheYear', 'position': 3, 'score': '', 'bucket': '', 'scroll_type': 'SS_Basic', 'title': 'Top Albums from Same Year', 'subtitle': '', 'highlight': '', 'simpleHeader': False, 'noHeader': False, 'source_api': True, 'source_params': {'album_year': '2023', 'album_lang': None}, 'view_more': [], 'is_JT_module': False}, 'artists': {'source': 'artists', 'position': 4, 'score': '', 'bucket': '', 'scroll_type': 'SS_Basic', 'title': 'Artists', 'subtitle': '', 'highlight': '', 'simpleHeader': False, 'noHeader': False, 'view_more': [], 'is_JT_module': False}}}
```
The `albumid` for the above album is `43600311`

Now that we have `albumid` (which I am saving in my beets plugin), I was hoping that we can use this `albumid` to get the same information. I tried manually creating an identifier using:
```python
id = {'type': 'ID', 'value': 43600311}
print(id)
print(jiosaavn.get_album_details(id))
```
The `id` looks fine and I tried different combinations, but none of them worked. I keep getting the error:
```
{'type': 'ID', 'value': '43600311'}
{'error': {'code': 'INPUT_MISSING', 'msg': 'One or more required field missing:albumid'}}
```

Basically, my question is how do we get album or song information using the albumid/songid that Saavn uses internally?
